source/loader/CMakeLists.txt

```
@@ -15,6 +15,24 @@ add_ur_library(ur_loader
     ${CMAKE_CURRENT_BINARY_DIR}/UrLoaderVersion.rc
 )
 
+if (MSVC)
+    set(TARGET_LIBNAME ur_loader)
+    string(TOUPPER ${TARGET_LIBNAME} TARGET_LIBNAME)
+    
+    set(LOADER_VERSION_SCRIPT ${CMAKE_CURRENT_BINARY_DIR}/ur_loader.def)
+    configure_file(${CMAKE_CURRENT_SOURCE_DIR}/loader.def.in ${LOADER_VERSION_SCRIPT} @ONLY)
+    set_target_properties(ur_loader PROPERTIES
+        LINK_FLAGS "/DEF:${LOADER_VERSION_SCRIPT}"
+    )
+else()
```

This should be checking if its Linux:

```suggestion
elseif(CMAKE_SYSTEM_NAME STREQUAL Linux)
```

---

Thanks, done.